docs/contributing/contributing.md

```diff
@@ -1,111 +1,158 @@
+At this point you should be ready to submit a Pull Request!  
+Check out our [Contributing Checklist](../contributing/checklist) to make sure you have all the parts you need.
 
-:::note XSOAR Technology Partners
-If you are an XSOAR Technology Partner, make sure you have reviewed the use cases with your [Cortex XSOAR Alliances Team](mailto:soar.alliances@paloaltonetworks.com) and that you have a *Partner ID* to associate your Pull Request to.
+:::tip
+[Hello World Content Pack](https://github.com/demisto/content/tree/master/Packs/HelloWorld) is a good example that implements all of the above guidelines and can be used as a reference (alongside the [Hello World Design Document](https://docs.google.com/document/d/1wETtBEKg37PHNU8tYeB56M1LE314ux086z3HFeF_cX0)).
 :::
 
-A good working example that summarizes all of the above is the [Hello World Content Pack](https://github.com/demisto/content/tree/master/Packs/HelloWorld) that you can use as a reference. Also check out the [Hello World Design Document](https://docs.google.com/document/d/1wETtBEKg37PHNU8tYeB56M1LE314ux086z3HFeF_cX0).
-
-This guide doesn't cover all the topics: please browse the left sidebar and use the search bar to find what you need, and reach out for help over [Slack](https://start.paloaltonetworks.com/join-our-slack-community) on the `#demisto-developers` channel when in doubt.
-
+:::note
+If you are an XSOAR Technology Partner, make the [Cortex XSOAR Alliances Team](mailto:soar.alliances@paloaltonetworks.com) has reviewed your planned contribution, and that you have a *Partner ID* to associate your Pull Request to.
+:::
 
-## Before opening a Pull Request
+## Pull Request Submission Process
+### Prerequisites
+Before submitting a Pull Request to the [Cortex XSOAR GitHub Repository](https://github.com/demisto/content), the following requirements must be met:
+1. Make sure to check the content contribution [checklist](../contributing/checklist), and assure that all the required files exist and are properly formatted.
+2. Create and upload your contribution to a GitHub **fork** of the XSOAR Content repository, and submit your changes **on a new branch** (do **not** push your changes to the *master* branch).
+3. Validate your content using [demisto-sdk validate](https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/validate/README.md):  
+    `demisto-sdk validate -i Packs/YourPackName`
+4. If your contribution has Integrations or Scripts, run linting (which also runs [unit-tests](../tutorials/tut-setup-dev#step-5-run-the-linter-and-unit-tests)) by running:  
+    `demisto-sdk lint -i Packs/YourPakName`
+5. Create a short demo video presenting the product and your pack, and link it.  
+    This will help our reviewers to better understand what the product is used for, and the newly contributed content integrates with it.
```

Comment:
```suggestion
    This will help our reviewers to better understand what the product is used for, and how the newly contributed content integrates with it.
```